core/tests/puppeteer-acceptance-tests/blogpostAdmin-tests/blog-post-delete-draft.js

```diff
@@ -0,0 +1,19 @@
+const e2eBlogPostAdmin = require("../utility-functions/blogPostAdminUtils.js");
```

Comment:
The dirname for this is as bit odd. I suggest blog-post-admin-tests instead of blogpostAdmin-tests.

Also, you can make the filenames match what's in the test. E.g. delete-draft-post, create-draft-post, etc. -- these are easy to understand. Otherwise you have this general "noun - verb" framing but I'm not sure if there's always a noun? With delete-draft-post there is this general idea of "blog post admin deletes draft post".

Lastly, it should be "blog admin" rather than "blog post admin" since the admin is in charge of the whole blog and not just individual blog posts.